andela/odin-ah-frontend
# #162158199 Implement management interface for user follow lists

#### What does this PR do?
- Implements a management interface for user follow lists

#### Description of Task to be completed?

- [x] Add follower and following containers
- [x] Add pagination
- [x] Refactor and restructure dashboard view to accommodate new UI elements
- [x] Add profile card
- [x] Add follow button container

#### How should this be manually tested?
- `git fetch`
- `git checkout ft-162158199-manage-follow-list`
- `npm install`
- `npm start`
- Create or login with an existing user
- Follow a few authors from the articles on the homepage
- Navigate to `/dashboard`
- You should see your following and follower count
- Click on the links to see the details of following and followers

#### What are the relevant pivotal tracker stories?
[#162158199](https://www.pivotaltracker.com/story/show/162158199)

#### Screenshots (if appropriate)
![screenshot_2018-11-29 authors haven](https://user-images.githubusercontent.com/30433120/49244667-a0d88a80-f410-11e8-81e8-a30e7086c05e.png)
